__tests__/AzureSqlAction.test.ts

```diff
@@ -132,6 +150,29 @@ describe('AzureSqlAction tests', () => {
         expect(getSqlPackagePathSpy).toHaveBeenCalledTimes(1);  // Verify build succeeds and calls into Publish
         expect(execSpy).toHaveBeenCalledTimes(2);
     });
+
+    describe('validate errors on unsupported sqlpackage action types', () => {
```

Comment:
I like how you're testing this negative scenario. How about another test for an invalid sqlpackage action as well? (e.g. Action 'foo')?

This would just ensure that we're handling invalid actions and giving users a consistent error message in future versions of the action

Reply:
Good point. I moved this test to main.test.ts as the error handling is mainly there. At this point it expects the SqlpackageAction enum type so if it's not one of the supported ones, it becomes undefined. The error handling is further upstream.